docs/cluster_configuration.md

```diff
@@ -94,10 +94,10 @@ spec:
 Once the update is made, the `<cluster-name>-pg` ConfigMap in the `ztp-<cluster-template-namespace>` namespace gets updated with the new value. This ConfigMap is used by the ACM policies in their hub templates.
 ```yaml
 $  oc get clustertemplate -A
-NAMESPACE                 NAME                   AGE
-sno-ran-du-v4-16   sno-ran-du.v1   3d23h
+NAMESPACE                 NAME                     AGE
+sno-ran-du-v4-Y-Z         sno-ran-du-v4-Y-Z-1      3d23h
```

Comment:
Should the NAME be `sno-ran-du.v4-Y-Z-1`? as the clustertemplate name is formatted as  <name.version>